### PR TITLE
fix/FOROME-1212: fix some button styles

### PR DESCRIPTION
**Pull request by QSdmitrioul**
[Task](https://quantori.atlassian.net/browse/FOROME-1212)

### CODE REVIEW

**Review by AndreyUstyumenko — APPROVED** [review]

File: src/components/undo-redo-buttons/components/undo-redo-button.tsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import style from '../undo-redo-buttons.module.css'
```
Comment (AlMaQntr) [inline src/components/undo-redo-buttons/components/undo-redo-button.tsx]: I think, you don't need UndoRedoButton component. It just a button with class

File: src/components/undo-redo-buttons/undo-redo-buttons.tsx [hunk 2]
```diff
@@ -18,17 +21,21 @@ export const UndoRedoButtons = ({
...
+      onClick={onRedo}
+      disabled={isRedoDisabled}
+      className={style.undoRedo_button_right}
+    >
       <Icon name="Redo" />
     </UndoRedoButton>
 
-    <Link className="flex flex-wrap ml-3" to={Routes.Root}>
-      <Icon size={15} name="Close" className="text-white cursor-pointer" />
+    <Link className={cn(style.undoRedo_closeIcon)} to={Routes.Root}>
```
Comment (AlMaQntr) [inline src/components/undo-redo-buttons/undo-redo-buttons.tsx]: I think that close link is not a logical part of UndoRedo buttons, can we extract it

**Review by AlMaQntr — APPROVED** [review]

**Review by Lapk1n — APPROVED** [review]

File: src/components/undo-redo-buttons/components/undo-redo-button.tsx (hunk 1 above)
Comment (QSdmitrioul) [reply src/components/undo-redo-buttons/components/undo-redo-button.tsx]: Done

File: src/components/undo-redo-buttons/undo-redo-buttons.tsx (hunk 2 above)
Comment (QSdmitrioul) [reply src/components/undo-redo-buttons/undo-redo-buttons.tsx]: Done

**Pull request merged**